messages/context.json

```
@@ -193,5 +193,6 @@
   "admin/editor.product-description.title-prop.title": "Title",
   "admin/editor.search-bar.title": "Search bar",
   "admin/editor.search-bar.placeholder.title": "Placeholder",
-  "store/back-to-top.label": "Back to top"
+  "store/back-to-top.label": "Back to top",
+  "store/search.input-label": "Label of search input"
```

```suggestion
  "store/input-search.label": "Search for products"
```

It's important to keep the name of the component as a pattern, the component is called InputSearch